framework/PostProcessors/ComparisonStatisticsModule.py

```diff
@@ -472,64 +468,49 @@ def collectOutput(self, finishedJob, output):
       for dataPull, data in zip(dataPulls, datas):
         dataStats, cdfFunc, pdfFunc = _getPDFandCDFfromData(str(dataPull),
                                                             data,
-                                                            csv,
+                                                            output,
                                                             self.methodInfo,
                                                             self.interpolation,
-                                                            generateCSV)
+                                                            True)
         self.raiseADebug("dataStats: " + str(dataStats))
         graphData.append((dataStats, cdfFunc, pdfFunc, str(dataPull)))
       graphDataDict = _getGraphs(graphData, self.fZStats)
-      if generateCSV:
-        for key in graphDataDict:
-          value = graphDataDict[key]
-          if type(value).__name__ == 'list':
-            utils.printCsv(csv, *(['"' + l[0] + '"' for l in value]))
-            for i in range(1, len(value[0])):
-              utils.printCsv(csv, *([l[i] for l in value]))
+      for key in graphDataDict:
```

Comment:
why is this part is kept since we do not generate a csv?

Reply:
we generate a csv

Reply:
my bad: I misread the manual file